metric_learn/itml.py

```diff
@@ -273,6 +273,7 @@ class ITML_Supervised(_BaseITML, TransformerMixin):
            be removed in 0.6.0.
   num_constraints: int, optional
       number of constraints to generate
+      (`20 * num_classes**2` constraints by default)
```

Comment:
We may need to use double-backticks here to prevent the docs from interpreting this as a link target. Can you build the HTML docs with these changes to verify?

Reply:
I tried it locally and I see it works as expected, also there is a `itml_supervised.fit(X, y, bounds=...)` that does not uses double backticks in the code a few lines below.

Should I make a pull request for the documentation that I generated as well ?
The commits that update the documentation [look a bit heavy for minor modifications](https://github.com/scikit-learn-contrib/metric-learn/commit/76ba4b30947fd288c2d4bfe9b496cb185b2760a3) and include some system-specific results.

Reply:
I tried it locally and I see it works as expected, also there is a `itml_supervised.fit(X, y, bounds=...)` that does not uses double backticks in the code a few lines below.

Should I make a pull request for the documentation that I generated as well ?
The commits that update the documentation [look a bit heavy for minor modifications](https://github.com/scikit-learn-contrib/metric-learn/commit/76ba4b30947fd288c2d4bfe9b496cb185b2760a3) and include some system-specific results.